src/components/admin/loading.tsx

```diff
@@ -0,0 +1,31 @@
+import { useTimeout, useTranslate } from "ra-core";
+import { Spinner } from "./spinner";
+
+export const Loading = (props: LoadingProps) => {
+  const {
+    loadingPrimary = "ra.page.loading",
+    loadingSecondary = "ra.message.loading",
+    ...rest
+  } = props;
+  const oneSecondHasPassed = useTimeout(1000);
```

Comment:
Can you make the delay configurable?

Reply:
Ok, I'll add a reminder to do it on the mui version too.